example/ipsec_crypto/odp_ipsec_stream.c

```diff
@@ -425,6 +425,11 @@ odp_bool_t verify_ipv4_packet(stream_db_entry_t *stream,
 	/* Find IPsec headers if any and compare against entry */
 	hdr_len = locate_ipsec_headers(ip, &ah, &esp);
 
+	/* Report failure if the stream is not IPsec protected as configured */
```

Comment:
The change looks otherwise ok to me but the comment does not seem to quite match with the test (the input part). Maybe reword "...if the packet is not IPsec encapsulated or cleartext as expected"?

Reply:
Agreed. Updating the comment as:
```
+       /* Verify if the packet is IPsec encapsulated or is cleartext as         
+        * expected                                                              
+        */
```
Let me know if this text looks OK.